src/containers/InputContainer.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+// styled component를 사용한다
+
+const AppInput = styled.div`
+  width: 28rem;
+  margin-bottom: 1rem;
+`;
+
+const FormControl = styled.form`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+`;
```

Comment:
`flex-direction: row;`는 디폴트 값이기 때문에 따로 설정하지 않으셔도 됩니다

```suggestion
const FormControl = styled.form`
  display: flex;
  justify-content: space-between;
  width: 100%;
`;
```

Reply:
저번과 같은 부분이군요, 다시 확인하도록 할게요!